test/create-flows/more_flows_shim.mlir

```diff
@@ -0,0 +1,93 @@
+//===- more_flows_shim.mlir ------------------------------------*- MLIR -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2021 Xilinx Inc.
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Because of the differences depending on the shim tiles, can you comment these test cases more?